frontend/src/components/TilePicker/TilePicker.tsx

```diff
@@ -22,7 +23,14 @@ export function TilePicker<T>() {
           <li
             className={'Tile' + (tile === this.props.value ? ' -Active' : '')}
             key={typeof tile.name === 'string' ? tile.name : tile.name.toString()}
-            onClick={() => this.props.onChange(tile)}
+            onClick={() => {
+              ReactGA.event({
+                category: 'Dataset',
+                action: 'Selected a dataset',
```

Comment:
is it best practice to use spaces? or should we use kebab-case?

Reply:
yes